sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
dual licence with BSD 3c

Reply:
@lgirdwood we decided to keep this file GPL2.0 onkly IIRC because of debugfs. @plbossart can you please correct me if I am wrong?

Reply:
@lgirdwood this was an explicit decision between @ranj063 and me.

a) the code extends the GPL-2.0-only auxiliary bus and uses the fundamentals of the device model (put_device, etc). Our experience with this type of code pushes both of us to recommend GPL-2.0-only.

b) a non-Linux implementation would have to redo everything anyways, so there are limited benefits to the dual license.

Reply:
I will leave them as GPL-2.0-only for now